advisories/staging/BRSA-zqgip7w0.toml

```diff
@@ -0,0 +1,16 @@
+[advisory]
```

Comment:
nit: could you update this commit message to be more specific? maybe "stage BRSA for containerd CVE-2024-47108"